src/assets/README.md

```diff
@@ -0,0 +1,178 @@
+# Asset Handling
+
+The ILDH uses [Laravel Mix](https://laravel-mix.com) as a wrapper for webpack to process JavaScript and CSS,
+[Sass](http://sass-lang.com) or [Stylus](http://stylus-lang.com) stylesheets.
+
+## Scripts
+
+By default, the Mix [configuration file](../../webpack.mix.js) includes configuration for processing the three
```

Comment:
There aren't three javascript files in `/src/assets/scripts`. It would be better to use more general wording. Thanks.